### ISSUE TITLE
Jetty , dropwizard issue, metrics already exist

### ISSUE THREAD

**Issue by javahas** [dropwizard/dropwizard#3301]
jetty version: 9.4.28.v20200408
dropwizardv version=4.1.7
JDK 11

metrics-core-4.1.7.jar
metrics-healthchecks-4.1.7.jar
metrics-jetty9-4.1.7.jar
metrics-json-4.1.7.jar
metrics-jvm-4.1.7.jar
metrics-servlets-4.1.7.jar

Just upgraded  version of dropwizard and jetty, now I get the following error when starting my application:

```
Caused by: java.lang.IllegalArgumentException: A metric named org.eclipse.jetty.servlet.ServletContextHandler.percent-4xx-1m already exists
	at com.codahale.metrics.MetricRegistry.register(MetricRegistry.java:156)
	at com.codahale.metrics.jetty9.InstrumentedHandler.doStart(InstrumentedHandler.java:143)
	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:72)
	at org.eclipse.jetty.util.component.ContainerLifeCycle.start(ContainerLifeCycle.java:169)
	at org.eclipse.jetty.util.component.ContainerLifeCycle.doStart(ContainerLifeCycle.java:117)
	at org.eclipse.jetty.server.handler.AbstractHandler.doStart(AbstractHandler.java:97)
	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:72)
	at org.eclipse.jetty.util.component.ContainerLifeCycle.start(ContainerLifeCycle.java:169)
	at org.eclipse.jetty.server.Server.start(Server.java:407)
	at org.eclipse.jetty.util.component.ContainerLifeCycle.doStart(ContainerLifeCycle.java:110)
	at org.eclipse.jetty.server.handler.AbstractHandler.doStart(AbstractHandler.java:97)
	at org.eclipse.jetty.server.Server.doStart(Server.java:371)
	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:72)
```

**Comment by joschi** [comment]
@javahas I'm unable to reproduce this with a new empty project and dropwizard-example based on Dropwizard 2.0.9 (which includes the mentioned versions of Dropwizard Metrics and Jetty).

Please provide a minimal project to reproduce your issue.

**Comment by joschi** [comment]
@javahas Better late than never.

This has been addressed in Metrics 4.1.14 (https://github.com/dropwizard/metrics/pull/1685) which is included in Dropwizard 2.0.14:
https://github.com/dropwizard/dropwizard/releases/tag/v2.0.14
